### DISCUSSION TITLE
Does Fred support RedisJson Module?

### DISCUSSION THREAD

**Discussion by tausifcreates** [aembke/fred.rs#168] [Q&A] [answered]
I noticed in the readme there was a mention of optional json support, but is it just storing json strings?

**Comment by aembke** [comment, marked as answer]
Hi @tausifcreates. You can use the `custom` or `custom_raw` functions to issue `JSON` commands now, but I'm also in the process of adding better support for this in version 7. That should be ready in the next few weeks if you'd prefer to wait. 

Update: [7.0.0](https://docs.rs/fred/7.0.0/fred/interfaces/trait.RedisJsonInterface.html) has an interface for this now.

**Reply by tausifcreates** [reply]
Hi, thanks for your reply. Fred has been of great help for me. Wish you the best.
